golang/cosmos/x/swingset/client/cli/tx.go

```diff
@@ -220,9 +229,11 @@ func GetCmdWalletAction() *cobra.Command {
 	return cmd
 }
 
+// NewCmdSubmitCoreEvalProposal is the CLI command for submitting a "CoreEval"
+// governance proposal via `agd tx gov submit-proposal swingset-core-eval ...`.
 func NewCmdSubmitCoreEvalProposal() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "swingset-core-eval [[permit.json] [code.js]]...",
+		Use:   "swingset-core-eval <permit.json code.js>...",
```

Comment:
I'm not sure of the correct placement of angle brackets to indicate that these are pairs of filenames.  Your rendition seems okay, but I wish there was something even clearer.